src/Makefile

```diff
@@ -10,7 +12,7 @@ VOSK_SOURCES= \
     spk_model.cc \
     vosk_api.cc
 
-CFLAGS=-g -O2 -std=c++17 -fPIC -DFST_NO_DYNAMIC_LINKING -I. -I$(KALDI_ROOT)/src -I$(OPENFST_ROOT)/include -I$(OPENBLAS_ROOT)/include
+CFLAGS=-g -O2 -std=c++17 -fPIC -DFST_NO_DYNAMIC_LINKING -DHAVE_CUDA=1 -I. -I$(KALDI_ROOT)/src -I$(OPENFST_ROOT)/include -I$(OPENBLAS_ROOT)/include -I$(CUDA_ROOT)/include -L$(CUDA_ROOT)/lib64
```

Comment:
-L flag belongs to LIBS together with cuda libraries, otherwise good

Reply:
Moved to LIBS + added a comment regarding additional CUDA libs location.